frontend/src/main/pages/RiderDashboard/RideDashboardIndexPage.js

```diff
@@ -0,0 +1,13 @@
+import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+
+export default function PlaceholderIndexPage() {
+
+  // Stryker disable all : placeholder for future implementation
```

Comment:
There shouldn't need to be a Stryker exception here.    If removing this causes you to need to implement some tests, then you need to implement those tests.